client/core/simnet_trade.go

```diff
@@ -1344,18 +1374,19 @@ func (s *simulationTest) checkAndWaitForRefunds(ctx context.Context, client *sim
 		return nil
 	}
 	if furthestLockTime.IsZero() {
-		client.log.Infof("No refunds necessary")
+		s.log.Infof("No refunds necessary for client %s.", client.name)
 		return nil
 	}
 
-	client.log.Infof("Found refundable swaps worth %s %s and %s %s", s.base.valFmt(refundAmts[s.base.id]),
+	s.log.Infof("Found refundable swaps worth %s %s and %s %s.", s.base.valFmt(refundAmts[s.base.id]),
```

Comment:
Are these `client.log -> s.log` changes just for the color? Seems like you could have just made the `client.log` a colored logger.

Reply:
No. They both log colored.... I didn't really mean to change these, will revert to what they were before.

Reply:
Hopefully got them all https://github.com/decred/dcrdex/compare/8003b1c6863c8c9200e33752ad96f1633f9754fc..8502a9c37e7204186ff1fbcdad8ff5de85e405f4